src/Connection/Socket/Handshake.php

```diff
@@ -73,12 +75,56 @@ public function __construct(string $username, string $password, int $version)
         $this->version = $version;
     }
 
+    /**
+     * @inheritdoc
+     * @throws \RuntimeException
+     * @throws Exception
+     */
+    public function helo(StreamInterface $stream): void
```

Comment:
Typo in hello?

Reply:
SMTP HELO command, I will change to Hello, makes more sense indeed since we do nothing with SMTP!